File: 2441. Largest Positive Integer That Exists With Its Negative.py
class Solution:
    def findMaxK(self, nums: List[int]) -> int:
        result = []
        for i in nums:
            if((-i) in nums):
                #print(i,-i)
                result.append(i)
        #print(result)
        if(len(result)):
            return max(result)
        return -1
        

File: 2441. Largest Positive Integer That Exists With Its Negative.cpp
class Solution {
public:
    int findMaxK(vector<int>& nums) {
        vector<int> result;
        for(int i=0;i<nums.size();i++){
            for(int j=0;j<nums.size();j++){
                if(nums[i] + nums[j] == 0){
                    result.push_back(nums[i]);
                }
            }
        }
        if(result.size() != 0){
            return *max_element(result.begin(),result.end());
        }
        
        return -1;
        
    }
};
